projects/angular-proto-firestore/README.md

```diff
@@ -0,0 +1,37 @@
+# Protobin reader template
+
+To set up your development environment, follow these steps:
+
+## Installation
+Install these:
+* [node](https://nodejs.org/) version 8.11.4  
+* [npm](https://www.npmjs.com/) version 5.6.0  
+* [google-protobuf](https://github.com/protocolbuffers/protobuf/releases), version 3.6.1  
+
+(Installation of exact versions is not required, but recommended. We can't guarantee that webapp will work properly with different versions.)
+
+Optional:
+* [firebase](https://firebase.google.com/docs/hosting/quickstart), to be able to deploy.  
+* [angular](https://angular.io/), to use `ng` features.  
+
+Run `npm install` to install npm modules  
+
+Run `npm run protoc` to generate proto functions.  
+
+## Running the server locally
+Run `npm start` to start a dev server. Navigate to `http://localhost:4200/`  
+Run `npm run build` to make a build.  
+Run `npm run deploy` to publish the app on the server.  
+Run `npm run functions` to update cloud functions on the server.  
+
+How to use firebase hosting:  
```

Comment:
I don't know...
Maybe their github: https://github.com/angular/angularfire2/tree/master/docs

But you think we should add the information?
It might be useful only for other developers, who didn't work with angularfire before.
Following the logic, we need to add docs for each technology: angular, material, css, html, gulp, protobuf, etc...

Reply:
Well, maybe just AngularFire and protobuf. The rest are pretty standard.
Only if there's a good source you know of.